### ISSUE TITLE
Remove TagHelper package reference from MVC samples as per Announcement #161

### ISSUE THREAD

**Issue by mikes-gh** [aspnet/Mvc#4450]
MVC Samples should react to this change

https://github.com/aspnet/Announcements/issues/161

By removing TagHelper references as this is now bought in transitively.
I was just looking at the project.json for another reason and noticed it.

**Comment by javiercn** [comment]
Closing this as it has already been cleaned up.
```
src\Microsoft.AspNetCore.Mvc\project.json:27:    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*",
test\Microsoft.AspNetCore.Mvc.TagHelpers.Test\project.json:10:    "Microsoft.AspNetCore.Mvc.TagHelpers": "1.0.0-*"
```

The only places where it appers are the above files. This is the query that I've used:
```Get-ChildItem -Recurse project.json | select-string "Microsoft\.AspNetCore\.Mvc\.TagHelpers"````

**Comment by dougbu** [comment]
Right, I fixed this w/ a60cce1 -- wasn't aware an issue existed or was assigned to @javiercn.

**Comment by dougbu** [comment]
Self-assigning since I did this trivial fix. My thanks to @javiercn for confirming I did all the projects.
